app/views/home/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<!-- Master Head -->
+<header class="master-head text-white text-center">
+  <div class="container">
+    <div class="row">
+      <div class="col-xl-9 mx-auto">
+        <h1 class="mb-5">Build and manage the next level surveys for your company!</h1>
+      </div>
+    </div>
+  </div>
+</header>
+
+<!-- Public Surveys -->
+<section class="packages text-center bg-light">
+  <div class="container">
+    <h2 class="mb-5">Fill any survey from our wide collection</h2>
+    <div class="row">
+      <!-- UPDATE: replace dummy i parameter with survey name and description parameters -->
+      <% @surveys.each do |i| %>
```

Comment:
do not use meaningless variables like here 'i' indicating index to me

Reply:
Mentioned the comment for its usage, However, replaced it now with the name to remove confusion